package.json

```diff
@@ -123,6 +127,11 @@
       "/test/sourcemaps/fixtures/lib"
     ]
   },
+  "commitlint": {
+    "extends": [
+      "@commitlint/config-conventional"
+    ]
+  },
```

Comment:
@watson looking at the [error msg on Jenkins](https://apm-ci.elastic.co/job/elastic+apm-agent-nodejs+pull-request+multijob-run-tests/54/NODEJS_VERSION=9,label=linux/console) could it be that this part here in `package.json` causes the issues already when trying to install npm modules?

Reply:
To me it seems that it's running `npm test` (which in turn will run the `test/lint-commits.sh` script). This happens after npm install. This is all expected, but what I intended was for the script [to discover](https://github.com/elastic/apm-agent-nodejs/pull/82/files#diff-f9ca946ada4c88c2435e8bcb5e555e60R5) that it's running inside Jenkins and not actually do anything - but maybe the `JENKINS` environment variable isn't set?